common/source/docs/common-arkflow.rst

```diff
@@ -0,0 +1,53 @@
+.. _common-arkflow:
+
+=====================================================
+ARK Flow Open Source Optical Flow and Distance Sensor
+=====================================================
+
+[copywiki destination="copter,plane"]
+
+The `ARK Flow Open Source Optical Flow and Distance Sensor <https://arkelectron.com/product/ark-flow/>`__ is an open source optical flow sensor including a Broadcom AFBR lidar which uses the CAN protocol to communicate with the autopilot.  This can be used to improve horizontal position control especially in GPS denied environments.
+
+.. image:: ../../../images/arkflow/ark_flow.jpg
+   :target: ../_images/arkflow/ark_flow.jpg
+   :width: 450px
+
+..  youtube:: SAbRe1fi7bU
+    :width: 100%
+
+Where to Buy
+------------
+
+The sensor is available from `ARK Electronics <https://arkelectron.com/product/ark-flow/>`__.
+
+Connection to Autopilot
+-----------------------
+
+- The ARK Flow is connected to the CAN bus using a Pixhawk standard 4 pin JST GH cable.
+- Multiple sensors can be connected by plugging additional sensors into the ARK Flow's second CAN connector.
+- The recommended mounting orientation is with the connectors on the board pointing towards back of vehicle, as shown in the following picture.
+
+.. image:: ../../../images/arkflow/ark_flow_orientation.png
```

Comment:
Hi, In the image, It looks like the aircraft is in NED frame and the sensor is in ENU frame, do we need to configure any orientation params, if we mount the sensor as mentioned in the image?

Reply:
You can ignore the arrows shown on the ARK Flow module in the picture. Those are just the CAD program coordinates. Not the sensor orientation. You want the CAN connectors on pointing towards the back/rear of the vehicle.